images/openwisp_nginx/Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM nginx:1.23.4-alpine
+FROM nginx:1.27.0-alpine
 
 RUN apk add --update --no-cache \
-            openssl~=3.0.15-r0 \
-            certbot~=1.32.0-r0 \
-            certbot-nginx~=1.32.0-r0 && \
+            py3-pip \
+            certbot \
+            certbot-nginx && \
```

Comment:
I removed the version specifiers from the system packages because there's only one version available to download from the Alpine repositories. Pinning dependencies leads to failing builds all the time with little to no benefits.

Reply:
> we have to choose between the risjk of a new incompatible version being released and them yanking an insecure release
I think we shouldn't start changing things now and leave it in an inconsistent state, I think is better to keep pinning now
if we want to change it, we should create an issue, plan it and execute it
not have some stuff unpinned and some pinned
it just makes everything more confusing

- @nemesifier from [OpenWISP Development chat](https://matrix.to/#/!lfWoNiqJKwUShxVcad:gitter.im/$wS6-2k70BJmzWwjCvVf8TumN3R8pcAVodmBJlJT8zLk?via=gitter.im&via=matrix.org&via=matrix.domainepublic.net)